samples/sensor/icp101xx/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _icp101xx:
+
+ICP101xx: Barometric Pressure and Temperature Sensor
+############################################
+
+Description
+***********
+
+This sample application periodically measures the sensor
+temperature and pressure, displaying the 
+values on the console along with a timestamp since startup.
+It also displays the estimated altitude.
```

Comment:
Should we talk about the timeout? As I understand the measurement stops after a determined time.

Reply:
No the measurement goes forever.
The timeout is because this sensor only has a trigger mode (the timeout is between the trigger and the data read).